src/celeritas/field/FieldDriverOptions.hh

```diff
@@ -67,7 +67,7 @@ struct FieldDriverOptions
         // clang-format off
       return  (minimum_step > 0)
 	       && (delta_chord > 0)
-	       && (delta_intersection > 0)
+	       && (delta_intersection > minimum_step)
```

Comment:
Hm, this update was while I was trying to come up with a simpler way of calculating the `miss_distance` (`FieldPropagator:193`). I think it's still a sensible bound, though, since for very small steps the chord and arc will be the same length, and then the start point of a substep will be "delta intersection" from the boundary.